components/automate-cds/server/server.go

```diff
@@ -7,29 +7,45 @@ import (
 	"google.golang.org/grpc/reflection"
 
 	grpc_s "github.com/chef/automate/api/interservice/cds/service"
+	profiles "github.com/chef/automate/api/interservice/compliance/profiles"
+	"github.com/chef/automate/components/automate-cds/config"
 	"github.com/chef/automate/components/automate-cds/service"
 	"github.com/chef/automate/lib/grpc/health"
 	"github.com/chef/automate/lib/tracing"
+	log "github.com/sirupsen/logrus"
 )
 
 // NewGRPCServer creates a grpc server that serves all Teams GRPC APIs
-func NewGRPCServer(s *service.Service) *grpc.Server {
+func NewGRPCServer(s *service.Service,
+	profilesServiceClient profiles.ProfilesServiceClient) *grpc.Server {
 	grpcServer := s.ConnFactory.NewServer(tracing.GlobalServerInterceptor())
 
 	health.RegisterHealthServer(grpcServer, health.NewService())
-	grpc_s.RegisterAutomateCdsServer(grpcServer, NewServer(s))
+	grpc_s.RegisterAutomateCdsServer(grpcServer, NewServer(s, profilesServiceClient))
 	reflection.Register(grpcServer)
 
 	return grpcServer
 }
 
 // GRPC creates and listens on grpc server
-func GRPC(addr string, s *service.Service) error {
-	list, err := net.Listen("tcp", addr)
+func GRPC(config config.AutomateCdsConfig, s *service.Service) error {
+	list, err := net.Listen("tcp", config.GRPC)
 	if err != nil {
 		return err
 	}
-	serv := NewGRPCServer(s).Serve(list)
-	s.Logger.Debugf("Automate CDS GRPC API listening on %v", addr)
+
+	// compliance-service Interface
+	complianceConn, err := s.ConnFactory.Dial("compliance-service", config.ComplianceService.Address)
+	if err != nil {
+		// This should never happen
+		log.WithError(err).Error("Failed to create compliance-service connection")
+		return err
+	}
+	defer complianceConn.Close() // nolint: errcheck
+
+	profilesServiceClient := profiles.NewProfilesServiceClient(complianceConn)
+
+	serv := NewGRPCServer(s, profilesServiceClient).Serve(list)
+	s.Logger.Debugf("Automate CDS GRPC API listening on %v", config.GRPC)
```

Comment:
Setting up the connection to the compliance-service and passing that connection to the cds server.